website/docs/docs/collaborate/access-from-dbt-cloud.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Access dbt Explorer from dbt Cloud features"
+sidebar_label: "Access from dbt Cloud"
+description: "Learn where and how to directly access and interact with dbt Explorer from dbt Cloud features and products."
+---
+
+<p style={{ color: '#808080', fontSize: '1.1em' }}>
+Access dbt Explorer from other features and products inside dbt Cloud, ensuring you have a seamless experience navigating between resources and lineage in your project. 
```

Comment:
This color might be a little hard to see for some folks using the dark background

<img width="793" alt="Screenshot 2024-06-11 at 2 10 00 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/60105315/0c9be306-6e84-47ab-9006-84ab2e6017d6">

Reply:
I use dark mode and it looked fine for me! I tested it via accessibility website and it passed but i can check again

Reply:
Is the point to accentuate this section? Darkening it feels like it puts it in the background rather than highlighting it

Reply:
I guess there's not harm leaving it as is, but as a reader, if I saw that (since we don't adopt that across many pages) my first thought would be "why is that grey?"

Reply:
for sure, it's meant to be like a sub intro and helpful for SEO too. i brought this up to the team [here](https://dbt-labs.slack.com/archives/C04559AB7D2/p1717585832860119) and thought everyone was onboard. however, i can bring this up in the docs meeting to further discuss, and am happy to remove it if you feel strongly about this.

Reply:
I think until we have some consistency across pages with it (instead of just this page) we should hold off on the color at least. If there's a way to implement the SEO optimization and make the color brighter, I'd be all for that. If we're going to implement the darker color, I think it would be better to do it across many pages at once, so this doesn't seem out of place.